src/sentry/static/sentry/app/plugins/components/issueActions.jsx

```diff
@@ -0,0 +1,333 @@
+import React from 'react';
+
+import {
+  Form,
+  FormState,
+  Select2Field,
+  Select2FieldAutocomplete,
+  TextareaField,
+  TextField
+} from '../../components/forms';
+import GroupActions from '../../actions/groupActions';
+import LoadingError from '../../components/loadingError';
+import LoadingIndicator from '../../components/loadingIndicator';
+import SettingsBase from '../../components/bases/settingsBase';
+import {t} from '../../locale';
+import {defined} from '../../utils';
+
+class IssueActions extends SettingsBase {
```

Comment:
yeah, agree, although it probably could be useful outside of plugins too. not sure of a good generic name, but I guess I can just use PluginComponentBase and if we wind up using it in other places, we can rename it